In the directory: /jukebox/LightSheetData/rat-brody/processed/201910_tracing/clearmap/z265/ClearMapCluster 
As the user: pnilsadmin 
on host: redshirt-n20 
Cpus_allowed_list:	0-6,16-22
Array Allocation Number: 18739718
Array Index: 0
In file included from /usr/people/pnilsadmin/.local/lib/python3.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1830:0,
                 from /usr/people/pnilsadmin/.local/lib/python3.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /usr/people/pnilsadmin/.local/lib/python3.7/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from /usr/people/pnilsadmin/.pyxbld/temp.linux-x86_64-3.7/pyrex/ClearMap/Analysis/VoxelizationCode.c:567:
/usr/people/pnilsadmin/.local/lib/python3.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
 #warning "Using deprecated NumPy API, disable it with " \
  ^

elastix is started at Tue Apr 21 14:40:17 2020.

which elastix:   elastix
elastix runs at: redshirt-n20
  Linux 3.10.0-1062.9.1.el7.x86_64 (x64), #1 SMP Fri Dec 6 14:55:59 EST 2019
  with 257752 MB memory, and 16 cores @ 1725 MHz.
-------------------------------------------------------------------------

Running elastix with parameter file 0: "/mnt/bucket/PNI-centers/LightSheetData/rat-brody/processed/201910_tracing/clearmap/z265/ClearMapCluster/parameterfolder/Order1_Par0000affine.txt".

Current time: Tue Apr 21 14:40:17 2020.
Reading the elastix parameters from file ...

ERROR: could not read fixed image.

itk::ImageFileReaderException (0x3c4d610)
Location: "unknown" 
File: /jukebox/pkgs/ITK/InsightToolkit-4.8.2/Modules/IO/ImageBase/include/itkImageFileReader.hxx
Line: 143
Description:  Could not create IO object for reading file /jukebox/LightSheetData/rat-brody/processed/201910_tracing/clearmap/z265/clearmap_cluster_output/cfos_resampled.tif
The file doesn't exist. 
Filename = /jukebox/LightSheetData/rat-brody/processed/201910_tracing/clearmap/z265/clearmap_cluster_output/cfos_resampled.tif



Errors occurred!
['run_clearmap_cluster.py', '3', '0']
0
setting elastixlib path as /jukebox/pkgs/ELASTIX/4.8/bin, ***note this is a hack and may fail***
Elastix sucessfully initialized from path: /jukebox/pkgs/ELASTIX/4.8
list index out of range
['elastix', '-m', '/jukebox/LightSheetData/rat-brody/processed/201910_tracing/clearmap/z265/clearmap_cluster_output/autofluo_for_cfos_resampled.tif', '-f', '/jukebox/LightSheetData/rat-brody/processed/201910_tracing/clearmap/z265/clearmap_cluster_output/cfos_resampled.tif', '-p', '/mnt/bucket/PNI-centers/LightSheetData/rat-brody/processed/201910_tracing/clearmap/z265/ClearMapCluster/parameterfolder/Order1_Par0000affine.txt', '-p', '/mnt/bucket/PNI-centers/LightSheetData/rat-brody/processed/201910_tracing/clearmap/z265/ClearMapCluster/parameterfolder/Order2_Par0000bspline.txt', '-out', '/jukebox/LightSheetData/rat-brody/processed/201910_tracing/clearmap/z265/clearmap_cluster_output/elastix_cfos_to_auto']
